xml/obs_build_containers.xml

```diff
@@ -91,4 +93,170 @@
+     compatible registry, with the URLs to the images constructed as follows:
+     <literal>
+       $BASE/$PROJECT/$REPOSITORY/$IMAGE_NAME:$TAG
+     </literal>
+     with the following components:
+     <itemizedlist>
+       <listitem>
+         <para>
+           <literal>$BASE</literal>: URL/IP under which the Open Build Service
+           instance is reachable
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$PROJECT</literal>: The name of the project where the image is
+           build, with all colons replaced with forward slashes.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$REPOSITORY</literal>: The name of the repository where the
+           containers are published.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$IMAGE_NAME</literal>: The name of this container image. It
+           defaults to the name of the package from which the container is
+           build. Alternatively, a different image name can be specified in the
+           build recipe, e.g. via the <literal>containerconfig</literal> element in a
+           &kiwi; file (see <link
+           xlink:href="https://osinside.github.io/kiwi/building_images/build_docker_container.html?highlight=containerconfig">the
+           kiwi documentation</link> for further details).
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           <literal>$TAG</literal>: The image tag. This defaults to
+           <literal>latest</literal> with alternatives being provided in a similar
+           fashion as the image's name.
+         </para>
+       </listitem>
+     </itemizedlist>
+   </para>
+   <para>
+     The <link
+     xlink:href="https://github.com/openSUSE/cooverview">cooverview</link>
+     project provides a simple user facing webpage to search for containers
+     published by the Open Build Service and can be used to conveniently obtain
+     the correct registry URLs. It is used to power <link
+     xlink:href="https://registry.opensuse.org/">registry.opensuse.org</link>
+   </para>
+ </sect2>
+
+ <sect3>
+   <title>
+     Container Image Signatures
+   </title>
+   <para>
+     The Open Build Service automatically signs every package that has been
+     build and publishes the cryptographic signature alongside with
+     it. Container images are no exception to this and the detached signatures
+     can be used by <link xlink:href="https://podman.io/">podman</link> to
+     verify every image that is pulled from the registry.
+   </para>
+   <para>
+     Podman has to be configured first as outlined in the following steps.
+     <itemizedlist>
+       <listitem>
+         <para>
+           Create a <literal>yaml</literal> file under
+           <literal>/etc/containers/registries.d/</literal> with an appropriate
+           name for your instance and the following contents:
+           <example>
```

Comment:
For layout improvements/simplicity, avoid nesting screens/examples/tables/figures/lists/... within `<para/>`. Instead close the para and start a new para later (i.e. on line 191):
```suggestion
         </para>
         <example>
```

This appears a few times below as well.